servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,35 @@
+---
+title: ServiceControl in Practice
+summary: Details how to think about the use of ServiceControl in production environments
+tags:
+- ServiceControl
+---
+
+ServiceControl is a tool that can accommodate many situations. Features can be enabled or disabled depending on the needs of the environment. Each feature is valuable in certain contexts but also comes with a cost in terms of resource usage and performance. This guidance is provided to provoke thought about why any given feature or plugin might be valuable in a system environment and to help decision makers make informed choices about how to get the best out of ServiceControl without providing solutions to specific scenarios that consider choices around hardware, transport or persistence.
+
+For example, in a development environment the [Debug Session](/servicecontrol/plugins/debug-session.md) and [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugins provide detailed information to support problem analysis. Particular advise that customers not use these  plugin outside of a development environment.
```

Comment:
How about:

For example, in a development environment the [Saga Audit](/servicecontrol/plugins/saga-audit.md) plugins provide detailed information such as tracking saga state changes to help debug a problem. However use of this plugin in a production environment can generate unnecessary load on SeriveControl and is not recommended.